integration-tests/integration_test.go

```diff
@@ -525,6 +553,75 @@ func (s *RepeatedNetworkFlowTestSuite) TestRepeatedNetworkFlow() {
 	fmt.Printf("ClientDetails from test: %s %s\n", s.clientContainer, s.clientIP)
 }
 
+// Launches nginx container
+// Launches gRPC server in insecure mode
+// Launches collector
+// Note it is important to launch the nginx container before collector, which is the opposite of 
+// other tests. The purpose is that we want ConnScraper to see the nginx endpoint and we do not want
+// NetworkSignalHandler to see the nginx endpoint.
+func (s *ConnScraperTestSuite) SetupSuite() {
+
+	s.metrics = map[string]float64{}
+	s.executor = NewExecutor()
+	s.StartContainerStats()
+	s.collector = NewCollectorManager(s.executor, s.T().Name())
+
+	s.collector.Env["COLLECTOR_CONFIG"] = `{"logLevel":"debug","turnOffScrape":` + strconv.FormatBool(s.turnOffScrape) + `,"scrapeInterval":2}`
+
+	s.launchNginx()
+
+	err := s.collector.Setup()
+	s.Require().NoError(err)
+
+	err = s.collector.Launch()
+	s.Require().NoError(err)
+
+	time.Sleep(10 * time.Second)
+
+
+	s.logLines = s.GetLogLines("grpc-server")
```

Comment:
Can't we get the grpc-server to store the necessary information in the BoltDB rather than looking in the logs?

Reply:
I am not sure. I tried doing that for the afterglow integration tests, but apparently BoltDB only stored one of the networking events. I know almost nothing about BoltDB, so maybe this is something that is actually easy, but looking at the grpc-server logs seemed like the best way of doing this.